src/store/state/state-model.ts

```diff
@@ -137,11 +137,7 @@ export const stateModel: StateModelIF = {
         legalType: '',
         identifier: ''
       },
-      nameRequest: {
-        legalType: null,
-        nrNumber: '',
-        legalName: ''
-      },
+      nameRequest: EmptyNameRequest,
```

Comment:
You should probably do `{ ...EmptyNameRequest }` so you're not pointing to the original object.

Reply:
dooh, will do!

Reply:
good ole javascript. Updated in next commit!